src/main/content/_i18n/en.yml

```diff
@@ -307,6 +296,4 @@ endofguide:
   star: Star
   download_sample_app: Download the sample application for this guide bundled with Open Liberty on github
   keep_exploring: Keep exploring
-  with_these_guides: with these guides.
-seo:
```

Comment:
![image](https://github.com/OpenLiberty/openliberty.io/assets/6392944/c1b2c7f4-101c-46cf-b262-6a6c70db2eec)
this translation is still used as well